### PR TITLE
Bug 1163550

### PR DESCRIPTION
**Pull request by seinlin**

### CODE REVIEW

File: aries.mk [hunk 1]
```diff
@@ -67,9 +67,9 @@ PRODUCT_COPY_FILES += \
     device/sony/aries/rootdir/system/etc/tfa98xx/VoiceCallEarpice_top.preset:/system/etc/tfa98xx/VoiceCallEarpice_top.preset \
     device/sony/aries/rootdir/system/etc/tfa98xx/VoiceCallEarpice_top.eq:/system/etc/tfa98xx/VoiceCallEarpice_top.eq
 
-PRODUCT_NAME := aosp_d5803
+PRODUCT_NAME := aries
```
Comment (lissyx) [inline aries.mk:17]: Can't we have a separate aries.mk that would include aosp_d5803 and just rewrite what we need ? Would make it easier to track upstream changes.

File: vendorsetup.sh [hunk 2]
```diff
@@ -15,3 +15,4 @@
 #
 
 add_lunch_combo aosp_d5803-userdebug
```
Comment (lissyx) [inline vendorsetup.sh]: Nit: I think it's wrong since we just removed aosp_d5803 from PRODUCT_MAKEFILES.

Comment (seinlin) [inline vendorsetup.sh]: Yes, we should remove this line even though there is no drawback for keeping.

File: aries.mk [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+PRODUCT_DEVICE := aries
+PRODUCT_MODEL := Xperia Z3 Compact (B2G)
+
+GAIA_DEV_PIXELS_PER_PX := 2.25
+BOOTANIMATION_ASSET_SIZE := 720p
+
+vendor_binary_files := $(strip $(wildcard vendor/sony/aries/aries-partial.mk))
+ifeq ($(vendor_binary_files),)
+  $(error Vendor binary files are not found. Please download from: \
+  http://developer.sonymobile.com/downloads/tool/software-binaries-for-aosp-lollipop-android-5-1/ \
```
Comment (lissyx) [inline aries.mk:27]: Should be better that we give the direct URL as http://dl-developer.sonymobile.com/eula/SW_binaries_for_Xperia_AOSP_Lollipop_MR1_v9_EULA.html

Comment (seinlin) [inline aries.mk:27]: I also try the Direct URL, but I notice that direct url will provide the fix version of release, such as v8 or v9, etc. The current used URL will always redirect to the latest release.

**Pull request closed without merging**